src/styles/_catalog.scss

```diff
@@ -1,10 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
   margin: 0 auto;
 
+  @media (min-width: 1024px) {
+    & {
+      grid-template-areas: "card card card card";
+    }
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
```

Comment:
If I understand it correctly, in today's lecture, Paul told us to write media queries separately but I am not sure. Maybe I misunderstood him.